app/assets/stylesheets/components/feedback_form.scss

```diff
@@ -5,3 +5,12 @@
     margin-top: 0;
   }
 }
+
+.general-feedback-form-group {
```

Comment:
I'm suprised custom CSS is required, using the design system correctly usually means you don't need to make any modifications out of the box.  I suspect that if you fix the fieldset issue below the CSS won't be required.